batchflow/models/torch/base.py

```diff
@@ -674,7 +676,7 @@ def initial_block(cls, inputs, **kwargs):
         torch.nn.Module or None
         """
         kwargs = cls.get_defaults('initial_block', kwargs)
-        if kwargs.get('layout'):
+        if kwargs.get('layout') or kwargs.get('base_block'):
```

Comment:
Initial block congifuration  ` initial_block/filters: 128` ( example from `torch_test.ipynb`) still will not go inside this `if` block

Reply:
What about moving `kwargs` checks inside `ConvBlock`?